src/conn/conn_layered.c

```diff
@@ -1191,32 +975,42 @@ __layered_drain_ingest_table(WT_SESSION_IMPL *session, WT_LAYERED_TABLE_MANAGER_
         } else if (WT_TIME_WINDOW_HAS_STOP(&tw))
             WT_ERR(__wt_upd_alloc_tombstone(session, &tombstone, NULL));
 
-        WT_ERR(__wt_upd_alloc(session, value, WT_UPDATE_STANDARD, &upd, NULL));
-        upd->txnid = tw.start_txn;
-        upd->start_ts = tw.start_ts;
-        upd->durable_ts = tw.durable_start_ts;
+        /*
+         * It is possible to see a full value that is smaller than or equal to the last checkpoint
+         * timestamp with a tombstone that is larger than the last checkpoint timestamp. Ignore the
+         * update in this case.
+         */
+        if (tw.durable_start_ts > last_checkpoint_timestamp) {
```

Comment:
Where did this come from? It's not obvious to me.

Reply:
The case is that we have a delete that is larger than the checkpoint timestamp and the base value smaller than or equal to the checkpoint timestamp. It means the delete is not included in the checkpoint but the full update is.